src/instructlab/model/train.py

```diff
@@ -281,6 +294,54 @@
     default=False,
     help="if true, enables the legacy linux training codepath from release 0.17.0 and prior.",
 )
+@click.option(
+    "--phased-training",
+    is_flag=True,
+    default=False,
+    help="Default: [False] Runs multi-phase training.",
+)
+@click.option(
+    "--phased-base-dir",
+    type=click.Path(),
+    default=lambda: DEFAULTS.PHASED_DIR,
+    help="Base directory for organization of end-to-end intermediate outputs.",
+)
+@click.option(
+    "--phased-phase1-data",
+    type=click.Path(),
+    help="Path to .jsonl file that will be used for the first phase of end-to-end training.",
+)
+@click.option(
+    "--phased-phase1-num-epochs",
+    type=int,
```

Comment:
No default?

Reply:
it can be int or None- the default for no-default is None, which is expected.